test/esIntegrations.test.js

```diff
@@ -228,6 +228,7 @@ describe('Testing ElasticSearch Integration', () => {
                   id: 4,
                   pub_place: 'Testtown',
                   items: [],
+                  publication_date: '[2020-01-01,2021-01-01)',
```

Comment:
Is this the format that'll be returned, then?

Reply:
No, it gets parsed by a regex to pull out that first year value, so `2020` will be output. (Edition dates will only every be single year ranges like I mentioned in standup